week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,11 +7,11 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {
-  return "$" + price * 1.3 * (1 + 1 / 100);
+function convertToUSD(priceInGBP) {
+  return price * 1.3 * 1.01;
 }
-function convertToBRL(price) {
-  return "R$" + convertToUSD(price) * 3.78328;
+function convertToBRL(priceInGBP) {
+  return price * 3.78 * 1.01;
```

Comment:
This is quite easy to understand and I like that. The question suggests you should think about DRY (don't repeat yourself) principals.

Imagine if tomorrow we wanted to change it to 2% charge on all currency conversions. We would need to to change it in two places. Maybe if we handle hundreds of currencies we would need to change it in hundreds of places.

As an extra improvement, can you find a way to use functions so that you'd only need to change the number in one place?